emscripten.py

```diff
@@ -639,15 +637,16 @@ def add_standard_wasm_imports(send_items_map):
       # Module scope, so lookup via Module as well.
       memory_import += " || Module['wasmMemory']"
     send_items_map['memory'] = memory_import
-    send_items_map['__indirect_function_table'] = 'wasmTable'
 
   # With the wasm backend __memory_base and __table_base are only needed for
   # relocatable output.
   if shared.Settings.RELOCATABLE:
-    send_items_map['__memory_base'] = str(shared.Settings.GLOBAL_BASE) # tell the memory segments where to place themselves
+    # tell the memory segments where to place themselves
+    send_items_map['__memory_base'] = str(shared.Settings.GLOBAL_BASE)
+    send_items_map['__indirect_function_table'] = 'wasmTable'
```

Comment:
oh, in relocatable mode do we still import the table then?

Reply:
Yes, for dynamic linking both the memory and the table come from JS and are exported the wasm.

For normal, static linking in theory they should be created and exported from the wasm side, like in standalone mode.   This change doesn't effect the memory, only the table through.